src/allmydata/_monkeypatch.py

```diff
@@ -9,12 +9,11 @@
 from __future__ import division
 from __future__ import print_function
 
-from future.utils import PY2
-if PY2:
-    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
-
 from warnings import catch_warnings
 
+from future.utils import PY2
+from util.future_builtins import *  # noqa: F401, F403
```

Comment:
Any wildcard import breaks used/unused flake detection so I don't think this is a viable approach.

Reply:
Upon further reflection: I think we can get away with this here because flake8 is not checking builtins, and those are the only things we're burying under the star import.